content/03.methods.md

```diff
@@ -23,15 +23,16 @@ Out of these 5 million articles, about 3 million were open access and available
 We downloaded a snapshot of this open access subset on January 31, 2020.
 This snapshot contains papers such as literature reviews, book reviews, editorials, case reports, research articles and more; however, we used only the research articles.
 
-### Comparing Corpora
-We used gensim [@raw:rehurek_lrec] (version 3.8.1) to preprocess the bioRxiv and PubMed Central corpora.
-We removed the 337 gensim-provided stopwords.
-Throughout our analysis we encountered non-word symbols (e.g., $\pm$), so we refer to words and symbols as tokens to avoid confusion.
+### Comparing Corpora
+We compared bioRxiv against Pubmed Central's Open Access corpus (PMCOA) and the New York Times Annotated corpus (NYTAC) [@raw:sandhaus2008new] to assess the similarities and differences between bioRxiv, PMCOA and NYTAC.
+Throughout our analysis we encountered non-word symbols (e.g., $\pm$), so we refer words and symbols as tokens to avoid confusion.
+We calculated the following statsitics for each corpus: the number of documents, the number of sentences, the total number of tokens, the number of stopwords, the average length of a document, the average length of a sentence, the number of negations, the number of coordinating conjunctions, the number of pronouns and the number of past tense verbs.
+Next, we used spaCy's "en_core_web_sm" model [@raw:spacy2] (version 2.2.3) to preprocess all corpora and filtered out 326 spaCy-provided stopwords.  

```

Comment:
Are these the same stopwords you use in the previous sentence?

Reply:
Yes these are the same stopwords that are used by spaCy.